src/main/java/com/github/fabienrenaud/jjb/stream/Serialization.java

```diff
@@ -165,4 +166,17 @@ public Object purejson() throws Exception {
         Value purejson = JSON_SOURCE().streamSerializer().purejson(JSON_SOURCE().nextPojo());
         return Generator.create().generate(purejson);
     }
+
+    @Benchmark
+    @Override
+    public Object antons() throws Exception {
+        sk.antons.json.JsonValue jso = JSON_SOURCE().streamSerializer().antons(JSON_SOURCE().nextPojo());
+
+        ByteArrayOutputStream baos = JsonUtils.byteArrayOutputStream();
+        Writer w = new BufferedWriter(new OutputStreamWriter(baos));
+        jso.writeCompact(w);
+        w.flush();
```

Comment:
Once you've removed the BufferedWriter, you can remove this explicit `flush` call too. `OutputStreamWriter.close()` will auto-flush the buffer.

Reply:
done